hasher-matcher-actioner/hmalib/common/tests/test_aws_dataclass.py

```diff
@@ -27,6 +27,11 @@ class Complicated:
     )
```

Comment:
Note we have renamed aws_dataclass_test → test_aws_dataclass to fit better with all other test modules.